pkgs/by-name/do/dotbot/package.nix

```diff
@@ -19,13 +20,17 @@ python3Packages.buildPythonApplication rec {
     patchShebangs bin/dotbot
   '';
 
+  patches = [
+    ./0001-fix-build.patch
```

Comment:
Comment: The purpose of the patch, whether having tracking PR, and when to remove it